csharp-basics/exercises/Polymorphism/Persons/Student.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Persons
+{
+    class Student: Person
+    {
+        public double gpa { get; set; }
```

Comment:
Vai tad sim nav jābūt private? un kur ir pārējie propertiji?

Reply:
šis varētu arī būt private.. bet pārējie props jau mantojas no Person klases.